### PR TITLE
Sbabu initialupdate

### PR DESCRIPTION
**Pull request by sak1997**
Added code of conduct and requirements files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# project-X
\ No newline at end of file
+# project-X
+
+# Hello! 
+
+### This is the initial commit for project-X! This is for Homework 2 for CSC 510. Welcome!
```
Comment (sak007) [inline README.md]: change to Homework 2b

Comment (sak1997) [reply README.md]: Done - please review and approve.

**Review by wizard-31 — APPROVED** [review]
Reviewed. Good to go.

**Review by sak007 — APPROVED** [review]
Good to go

**Pull request merged**